ENA_GBIF_Mapper/ena_gbif_mapper.py

```diff
@@ -0,0 +1,81 @@
+# std
+from dateutil.parser import parse
+import re
+import requests as requests
+
+# 3rd party
+import pandas as pd
```

Comment:
Not sure if you need pandas, could just use the default csvReader/Writer. I might be wrong, but I believe pandas pulls the whole csv in memory. If you use the default csvReader you could run over the csv streaming and stream to the output files. This would reduce the memory overhead of the application and allow to run over files of unlimited length.

Reply:
Yes, that's right. Thanks for your suggestion! I have replaced pandas with csvReader/Writer in all read/write parts.